code/game/objects/items/weapons/switchtool.dm

```diff
@@ -59,6 +62,9 @@
 		choose_deploy(user)
 
 /obj/item/weapon/switchtool/attackby(var/obj/item/used_item, mob/user)
+	if(istype(used_item, /obj/item/weapon/screwdriver) && no_remove)
```

Comment:
I made the `no_remove` var not just to prevent the removal of the item, but so it could give you that text if you tried to remove things from the tool. I will update the screwdriver check to be `removing_item` though. I had plans to make the tools removable and be transferred to another holotool handle thing so you could distribute them and I forgot to clean this up after I scarped the idea.